### DISCUSSION TITLE
[Feature Request] [pump] Invisible notetype parsing

### DISCUSSION THREAD

**Discussion by SheepyChris** [TeamRizu/OutFox#506] [03 - Feature Requests]
### Please Select the game mode your feature request is about.

pump

### Is your feature request related to a problem?

Certain charts such as Ignis Fatuus S22 from StepF2 utilizes the notetype as a special gimmick. Currently the game maps no notes whatsoever or visible notes to the wrong columns.

### Describe the solution you'd like

This has been added before as a proof of concept by Jose, however was scrapped in favor of a later rewrite as part of the new PIU NotesLoader.

**Comment by Scraticus** [comment]
Yep, h notes are done, s and v have work begun, and will be done for 4.16
